trackio/sqlite_storage.py

```diff
@@ -233,11 +220,38 @@ def get_metrics(project: str, run: str) -> list[dict]:
 
             return results
 
+    @staticmethod
+    def load_from_dataset():
+        dataset_id = os.environ.get("TRACKIO_DATASET_ID")
+        space_repo_name = os.environ.get("SPACE_REPO_NAME")
+        if dataset_id is not None and space_repo_name is not None:
+            hfapi = hf.HfApi()
+            if not TRACKIO_DIR.exists():
+                TRACKIO_DIR.mkdir(parents=True, exist_ok=True)
+            with SQLiteStorage.get_scheduler().lock:
+                try:
+                    files = hfapi.list_repo_files(dataset_id, repo_type="dataset")
+                    for file in files:
+                        if not file.endswith(".db"):
+                            continue
+                        downloaded_path = hf.hf_hub_download(
+                            dataset_id, file, repo_type="dataset"
+                        )
+                        shutil.copy(downloaded_path, TRACKIO_DIR)
```

Comment:
Instead of downloading and then copying, could we use the `local_dir` argument of `hf_hub_download`?

Reply:
Good call.